test/README.md

```diff
@@ -344,3 +348,18 @@ addition to regular error message.
 
 `NVIM_TEST_MAXTRACE` (U) (N): specifies maximum number of trace lines to keep. 
 Default is 1024.
+
+Troubleshooting
+---------------
+
+If `make test` fails early (e.g. because of `unknown target 'functionaltest'`),
+make sure the needed Lua rocks are installed. They are not automatically fetched
```

Comment:
Maybe use a more general term than `rocks` here (`module` ?).  The relevant part is that it's installed, not that it's installed using luarocks.